internal/test_helpers/scenarios/gleam/build/packages/gleam_json/src/gleam_json_ffi.mjs

```
@@ -0,0 +1,193 @@
+import { Ok, Error } from "./gleam.mjs";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)